### DISCUSSION TITLE
Agenda event scheduling requires choosing a space

### DISCUSSION THREAD

**Discussion by sergeByishimo** [exoplatform/exo-community#27] [Q&A]
It seems I cant schedule an event without specifying a space. It is a little bit of hassle to have to create a space when you just want to schedule a meeting event with just one other person.

Any luck this feature exists and I can not see it?

**Comment by rdenarie** [comment]
Hello @sergeByishimo 
You're right, it is no more possible to schedule an event outside of a space.
It was possible before, as each user had a personal calendar, but since few version, we rework the agenda application simplify it and to make it easier to use and administrate, and we remove the personal calendar.

So, there is no plan to reborn this possibility
